lib/paramedic.js

```diff
@@ -271,9 +271,13 @@ ParamedicRunner.prototype.runLocalTests = function () {
     })
     .then(function(command) {
         self.setPermissions();
-        logger.normal('cordova-paramedic: running command ' + command);
 
-        return execPromise(command);
+        if (!self.config.runMainTests()) {
```

Comment:
My bad, it actually works because the Android emulator gets started by `ParamedicTargetChooser`.
Then the aforementioned comment should also be deleted.